### ISSUE TITLE
Migration to GitHub is not complete

### ISSUE THREAD

**Issue by Neustradamus** [notepad-plus-plus/notepad-plus-plus#14454]
Dear @notepad-plus-plus team, @donho,

It will be nice to remove old parts like:
- https://sourceforge.net/p/notepad-plus/discussion/
- https://sourceforge.net/p/notepad-plus/patches/

There is a link to a dead docs website (to remove too):
- https://sourceforge.net/projects/notepad-plus/support

Can you add a "move" box like https://sourceforge.net/projects/handbrake for example.
- https://sourceforge.net/p/notepad-plus/

To disable/delete tabs, go here: https://sourceforge.net/projects/notepad-plus/
To add a new tab, go here: https://sourceforge.net/projects/notepad-plus/
To change informations, go here: https://sourceforge.net/p/notepad-plus/admin/overview
To export, go here: https://sourceforge.net/p/notepad-plus/admin/export

The new place can be added in:
- "Moved Project to", here: https://sourceforge.net/p/notepad-plus/admin/overview
- A new tab, go here: https://sourceforge.net/projects/notepad-plus/

Thanks in advance.

**Comment by Neustradamus** [comment]
Note: Need an update here too:
https://sourceforge.net/projects/gup-w32/

**Comment by Neustradamus** [comment]
@donho: I think you have not understood me, can you read my request?

You have not removed old parts:
- https://sourceforge.net/p/notepad-plus/discussion/
- https://sourceforge.net/p/notepad-plus/patches/

There is a link to a dead https://docs.notepad-plus-plus.org/ link here, please remove:
- https://sourceforge.net/projects/notepad-plus/support

You have not added the "move" box like https://sourceforge.net/projects/mumble for example (like a lot of other projects).

Can you reopen?

You have not done same for https://sourceforge.net/projects/gup-w32/.

Thanks in advance.

**Comment by donho** [comment]
@Neustradamus 
Remove from what then?
If you mean that remove some part from the project page of Notepad++ in SourceForge, I have to say it again:
"The project page of Notepad++ in SourceForge is no longer maintained."

The thing is, even I wanted to keep maintaining the project page of Notepad++ in SourceForge, I CANNOT login into sourceforge anymore with the account having the adm right. I try to the link "Forgot your password?" but I didn't get any email from SF (in the spam folder either).

**Comment by donho** [comment]
> I can done a ticket for you here for you:

Thank you, but it's not necessary, really.
Sorry for the repeating, but it seems that what I have said is not heard:
***"The project page of Notepad++ in SourceForge is no longer maintained."***

**Comment by alankilborn** [comment]
I think maybe the point is that the Sourceforget Notepad++ stuff should be taken down, so people aren't confused about where such information is hosted?

**Comment by donho** [comment]
@alankilborn 
> I think maybe the point is that the Sourceforget Notepad++ stuff should be taken down, so people aren't confused about where such information is hosted?

I don't think so. The official website of Notepad++ points to Github clearly. 
Though the page on SF is not maintained anymore, I think the historic of discussions of the project is interesting to be kept. 
So I prefer to keep it as it is. If SF want to take it down, they will do it.

**Comment by alankilborn** [comment]
> I think the historic of discussions of the project is interesting to be kept

I agree with this.

I was not agreeing with @Neustradamus , if his goal was indeed to get it taken down.  (Still not sure what his goals really are/were)

I was merely trying to force him into saying his goals.  :-)

**Comment by Neustradamus** [comment]
My main goal is to see the moved box like other projects and the removal of dead link here:
- https://sourceforge.net/projects/notepad-plus/support

**Comment by Neustradamus** [comment]
And there is a Notepad++ mirror too:
- https://sourceforge.net/projects/notepadplusplus.mirror/

I have not seen a mirror for gup-w32.

And it is always used:
- https://sourceforge.net/projects/npp-plugins/

**Comment by donho** [comment]
@Neustradamus 
> My main goal is to see the moved box like other projects and the removal of dead link here:
> 
>     * https://sourceforge.net/projects/notepad-plus/support

I won't bother with it.



> And there is a Notepad++ mirror too:
> 
>     * https://sourceforge.net/projects/notepadplusplus.mirror/

The mirror has nothing to do with legacy Notepad++ page on SF, but it's SF's stuff clearly.
